docs/posts/ibis-version-9.0.0-release/index.qmd

```diff
@@ -0,0 +1,457 @@
+ Snowflake, and more, from the comfort of your Python
+ repl.](https://us.pycon.org/2024/schedule/presentation/55/)
+- [Charlas: Porque SQL está en todas
+ partes.](https://us.pycon.org/2024/schedule/presentation/117/)
+
+::: {.callout-note}
+The schedule for SciPy 2024 is not available yet, keep an eye out for Ibis sessions.
+:::
+
+You can also listen to a [recent podcast episode with Real
+Python](https://realpython.com/podcasts/rpp/201) where Phillip Cloud, lead
+maintainer of Ibis, talks about the project.
+
+More podcasts and talks are in the works, so stay tuned!
+
+### New committers
+
+We're excited to welcome [Nick Crews](https://github.com/NickCrews) and [Naty
+Clementi](https://github.com/ncclementi) as the newest committers to the Ibis
+project!
+
+Nick has been a long-time contributor to
+Ibis, one of the most active people on the issue tracker, and increasingly
+involved in the community. We're excited to have him on board!
+
+Naty has been contributing to Ibis for a while now, leading the effort to add
+geospatial support to the DuckDB backend in addition to various other
+contributions. We're excited to have her on board!
+
+#### 9.0 contributors
+
+Ibis 9.0 had contributions from many people, including commits from:
+
+- Phillip Cloud
+- Krisztián Szűcs
+- Gil Forsyth
+- Jim Crist-Harif
+- Cody Peterson
+- Nick Crews
+- Naty Clementi
+- Chloe He
+- Nicola Coretti
+- Deepyaman Datta
+- Péter Gyarmati
+- Jiting Xu
+- Mehmet Fatih Aktas
+- Tyler White
+- Bryce Mecum
+- Riezebos
+- Chelsea Lin
+- Chip Huyen
+- Dan Lovell
+- Daniel Mesejo
+- Kexiang Wang
+- Mark Needham
+- Ray Bell
+- Thierry Jean
+- pieter-factful
+- saschahofmann
+
+Thank you to everyone who contributed to this release! And a special thanks to
+Krisztián Szűcs for his work on the internal representation and SQLGlot refactor
+work, it has drastically improved the Ibis codebase.
+
+With this refactor, it's never been easier to contribute to Ibis! Get on touch
+via [GitHub](https://github.com/ibis-project/ibis) or
+[Zulip](https://ibis-project.zulipchat.com/) if you're interested.
+
+## Backends
+
+No new backends were added in this release, [unless you count the April Fools'
+Day Unix backend](../unix-backend/index.qmd)! As always, there have been
+backend-specific improvements and bug fixes. Some highlights are below.
+
+Check the [full changelog](../../release_notes.qmd#9.0.0) for more details.
+
+### Snowflake
+
+You can now run Ibis inside of a Snowflake stored procedure! Look for a blog on
+this coming soon.
+
+### DuckDB
+
+In this release, `asof` joins are also now supported with a `tolerance`
+parameter.
+
+With [DuckDB's addition of MySQL as a database it can attach
+to](https://duckdb.org/2024/01/26/multi-database-support-in-duckdb.html), we've
+added a `read_mysql` function to Ibis inline with `read_postgres` and
+`read_sqlite`.
+
+For geospatial operations, casting from binary to geometry is now supported.
```

Comment:
adding something like: 
> This means that you can read a parquet file that has a geometry column in WKB format, and easily cast it to geometry file to do operations on it. 

pseudocode: 

```python 
table = ibis.read_parquet("your_parquet_file_with_wkb_geo_column")
table = table.mutate(geo_col=table.geometry.cast("geometry")) # table.geometry is the original column
```

Reply:
can we use `ibis.examples` for the data here?

Reply:
done w/ zones

Reply:
@lostmygithubaccount Not really, the zones example is from a table in `geojson` meaning it's not in binary it already reads in the correct format. If want to read some light data you can use this file. 

https://github.com/ibis-project/testing-data/raw/master/parquet/geo_wkb.parquet

```python
t = ibis.read_parquet("https://github.com/ibis-project/testing-data/raw/master/parquet/geo_wkb.parquet")

t.geometry #you'll notice it's binary

t = t.mutate(geo_col=t.geometry.cast("geometry"))

t.geo_col.cantroid() #not you can do geo operations
```